libdeno/binding.cc

```diff
@@ -246,6 +246,57 @@ v8::Local<v8::Object> DenoIsolate::GetBuiltinModules() {
   return handle_scope.Escape(builtin_modules_.Get(isolate_));
 }
 
+v8::ScriptOrigin ModuleOrigin(v8::Isolate* isolate,
+                              v8::Local<v8::Value> resource_name) {
+  return v8::ScriptOrigin(resource_name, v8::Local<v8::Integer>(),
+                          v8::Local<v8::Integer>(), v8::Local<v8::Boolean>(),
```

Comment:
I'd be a little more explicit to use `v8::Local<v8::Integer>(0)` for line/column offsets and `v8::False()` rather than relying on implicit defaults here.

Reply:
I think the defaults are wanted here. They are supplied only so the 9th argument "is_module" can be set. This code comes from the V8 tests.